.github/workflows/draft-release.yml

```diff
@@ -3,8 +3,6 @@ name: Draft Release
 on:
   push:
     branches: [ main ]
-  pull_request:
-    branches: [ main ]
```

Comment:
Hmm, okay. So if the PR is merged into main it would not show up in the release? Currently it is already showing up in the release draft even though it is not merged yet and might not will be.

Reply:
It will show up but it will not be categorized, as it has no label. See https://github.com/finleap-connect/monoskope/blob/b6f66b226ea0fbb1c631fb0d0366ea0ace74d871/.github/release-drafter.yml#L3

and I took a look at the current drafts and couldn't see this one (or any open pull-request). As far as I understood from the documentation open pull requests will not be considered in the draft (or at least the release).

therefore I suggest keeping this.

one more thing to consider (which ii discovered recently) is to allow release-drafter to release on tags. The benefit of this is when we, for example, want to jump from 3.5.0 to 4.0.0 all what we have to do is push the tag v4.0.0 and release-drafter will use the pushed tagged when releasing.

i have an example here where i use this:
https://github.com/HaniAlshikh/plugin.video.hs/blob/development/.github/workflows/draft-release.yml

https://github.com/HaniAlshikh/plugin.video.hs/blob/development/.github/workflows/release.yml

Reply:
I've seen that it works that way for monoctl funnily